File: exercicios/2061/2061.py
def entrada():
    x = input().split()
    aba = int(x[0])
    qt = int(x[1])
    return aba,qt

def main():
    aba,qt = entrada()

    for i in range(qt):
        o = input()
        if o == 'fechou':
            aba += 1
        
        if o == 'clicou':
            aba -= 1
   
    print(aba)
    
main()

File: exercicios/JavaScript/2061/2061.js
var input = require('fs').readFileSync('stdin','utf-8');
var lines = input.split('\n');

var x = lines.shift().split(' ');

var aba = parseInt(x[0])
var qt = parseInt([x[1]])

for(let i=0; i<qt; i++){
    var o = lines.shift()

    if(o == 'fechou'){
        aba += 1 
    }
    if(o == 'clicou'){
        aba -= 1
    }
}
console.log(aba)